__init__.py

```diff
@@ -55,12 +55,12 @@ class OWMApi(Api):
 
     def __init__(self):
         super(OWMApi, self).__init__("owm")
-        self.lang = "en"
+        self.owmLang = "en"
```

Comment:
We try to follow the pep8 non-camel-case variable name convention (classes camel case, globals all CAPS, local variables and class members underscored)